ci-operator/step-registry/baremetalds/e2e/ipv4/baremetalds-e2e-ipv4-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+workflow:
+  as: baremetalds-e2e-ipv4
+  steps:
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        NETWORK_TYPE=OpenShiftSDN 
+    pre:
+      - chain: baremetalds-ipi-pre
+    test:
+      - ref: baremetalds-e2e-conf
```

Comment:
I think that a useful preliminary task could be to define a chain for the `test` part, to increase the reuse amongst the different specific workflows we're going to define. 
As an additional point, it would be nice to remove `baremetalds-e2e-conf` from that chain (we've done some previous work to allow this removal, worth to be re-checked if it's fine now)

Reply:
Just curious, if we remove the `baremetalds-e2e-conf`, we will only be left with the `baremetalds-e2e-test` ref. Won't this be redundant if we create a chain with only one ref inside it?

Reply:
The idea is to prepare the ground for future extensions: in case it would be required it will be easy to add another step just in the chain without touching the workflow. This will also help those jobs that would like to override the `pre` chain: in that case calling the  `test` and `post` chain will ensure the job author that all the eventual changes done by baremetalds maintainers will flow in correctly.